pkg/options/types.go

```diff
@@ -130,6 +130,9 @@ func (n *NamespaceList) Type() string {
 	return "string"
 }
 
+// LabelWildcard allowlists any label
```

Comment:
Do you mind adding a few test cases for the `TestLabelsAllowListSet` test in types_test.go file.